content/en/admin/config.md

```diff
@@ -808,6 +808,16 @@ It is important to use a supported file format (JPEG or PNG, not SVG).
 
 ## Limits {#limits}
 
+### Anti Spam / Abuse 
+
+#### `HCAPTCHA_SITE_KEY`
+#### `HCAPTCHA_SECRET_KEY`
+
+Set this to your hCaptcha site key and secret to enable a checkbox in your administration panel allowing you to enable captchas on the account confirmation page using hCaptcha.
+
+Key can be obtain by registering on [https://www.hcaptcha.com/](https://www.hcaptcha.com/)
```

Comment:
Once #1406 would suggest cross-linking to this page to explain setup process.

Reply:
agreed, let's get #1406 added and then add a link here.